<?xml version="1.0"?>
<launch>
    
    <group ns="control">

        <!-- Follower -->

        <node pkg="practice_6" type="pure_pursuit_follower.py" name="pure_pursuit_follower" output="screen" required="true" >
            <remap from="path" to="/planning/local_path" />
        </node>

        <rosparam command="load" file="$(find practice_6)/config/control.yaml"/>

    </group>

</launch>